### DISCUSSION TITLE
Reset button and ET

### DISCUSSION THREAD

**Discussion by tigrrrlily** [JaapvanEkris/openrowingmonitor#22] [Q&A]
On v1beta-updates, pressing the reset button clears all fields including Elapsed Time, but upon workout start the ET starts counting from wherever it was before the reset. Is there a way of actually resetting ET so that my wife doesn't have to use her phone as a timer?

I'm *really* excited about this project btw in case you can't tell, the v1beta-updates branch has finally given me the functionality to make the hardware part of the project worth it.

**Comment by JaapvanEkris** [comment]
Sounds like a bug. I'll look into it and see what goes wrong. Thanks for reporting!

**Comment by JaapvanEkris** [comment]
I think this was indeed a bug. I've iproved the reset behaviour, if all goes well, it should work after updating your install.
